lib/protocol-decoder.js

```diff
@@ -59,9 +60,16 @@ function protocolDecoder (client, data) {
     var headers = client.req && client.req.headers ? client.req.headers : null
     var proxyProto
     if (headers) {
-      if (headers['x-real-ip']) proto.ipAddress = headers['x-real-ip']
-      else if (headers['x-forwarded-for']) proto.ipAddress = headers['x-forwarded-for']
+      if (headers['x-forwarded-for']) {
+        var addresses = forwarded(client.req)
```

Comment:
Could we extract the addresses without the use of the `forwarded` package ?

Reply:
Yes sure, but why ?
As you can see [forwarded](https://github.com/jshttp/forwarded/blob/master/index.js) is a very small module.

Reply:
In my opinion keep forwarded is ok